server/controllers/auth.controller.js

```diff
@@ -10,16 +9,21 @@ const activUser = require('../../config/activUserConfig.json').activUser;
 
 module.exports = {
   login(req, res) {
-    User.findOne({where: {email: req.body.email}}).then(user => {
-      if (!user || !passwordHash.verify(req.body.password, user.password)) {
-        res.status(401).send('Email or password is not valid');
-      } else {
-        const token = jwt.sign({
-          id: user.id
-        }, secret.key, {expiresIn: '2h'});
-        res.json({'token': token, 'user_id': user.id});
+    let token;
+    let id;
+    User.findOne({
+      where: {
+        email: req.body.email
       }
     })
+    .then(user => {
+      (!user || !passwordHash.verify(req.body.password, user.password)) &&
```

Comment:
can you create function  which will be check (!user || !passwordHash.verify(req.body.password, user.password))  and return true or false